lib/index.js

```diff
@@ -63,24 +63,26 @@ class Maid {
         throw new MaidError(`Task '${task.name}' failed.\n${err.stack}`)
       }
       if (checkTypes(task, ['sh', 'bash'])) {
-        return runCLICommand({ task, resolve, reject })
+        runCLICommand({ task, resolve, reject })
+        return
```

Comment:
why split the `return` expression?

Reply:
because it's more meaningful and it may be a bit confusing. we dont return promise from there anyway. and we dont need to return from there either because we are passing resolve and reject to it.

Reply:
also, return frome promise constructor make no sense, you are forced to use resolve and reject anyway. only if something fail in that constructor only then it goes to the catch handler. 

in this case it will just throw , because we directly await the promise